java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/WindowAggPlan.scala

```diff
@@ -730,7 +730,7 @@ object WindowAggPlan {
 
 
 
-  def isValidOrder(key: Long): Boolean = {
+  def isValidOrder(key: java.lang.Long): Boolean = {
```

Comment:
`isValidOrder` always check the orderKey from `val orderKey = computer.extractKey(row)`.
But `extractKey` returns scala Long, you should make it return java Long too.

Reply:
> There are some test failures. Please make sure you can run tests in local @Ivyee17 .

I've already made changes in the previous PR, it will cause test failures, as @dl239 said before.

Reply:
The failure is about delete-extract-key, if changing it to java.lang.Lang too, then this test will fail. Searching a key which is deleted before will be true, not false. I wonder if there's mistakes in `extractKey` function. Besides, I do think it's far beyond what should be done in this PR. I just need to resolve the warning log, not the complete process, right? Another issue and pr should be taken instead of this issue!!